src/proposals/20250529/SparkEthereum_20250529.t.sol

```diff
@@ -161,6 +165,26 @@ contract SparkEthereum_20250529Test is SparkTestBase {
         IRateLimits       rateLimits = IRateLimits(Optimism.ALM_RATE_LIMITS);
         ForeignController controller = ForeignController(Optimism.ALM_CONTROLLER);
 
+        assertEq(almProxy.hasRole(almProxy.CONTROLLER(),     Optimism.ALM_CONTROLLER), false, "incorrect-controller-almProxy");
+        assertEq(rateLimits.hasRole(rateLimits.CONTROLLER(), Optimism.ALM_CONTROLLER), false, "incorrect-controller-rateLimits");
+        assertEq(controller.hasRole(controller.FREEZER(),    Optimism.ALM_FREEZER),    false, "incorrect-freezer-controller");
+        assertEq(controller.hasRole(controller.RELAYER(),    Optimism.ALM_RELAYER),    false, "incorrect-relayer-controller");
+
+        _assertRateLimit(RateLimitHelpers.makeAssetKey(controller.LIMIT_PSM_DEPOSIT(), Optimism.USDC),   0, 0);
+        _assertRateLimit(RateLimitHelpers.makeAssetKey(controller.LIMIT_PSM_WITHDRAW(), Optimism.USDC),  0, 0);
+        _assertRateLimit(RateLimitHelpers.makeAssetKey(controller.LIMIT_PSM_DEPOSIT(), Optimism.USDS),   0, 0);
+        _assertRateLimit(RateLimitHelpers.makeAssetKey(controller.LIMIT_PSM_WITHDRAW(), Optimism.USDS),  0, 0);
+        _assertRateLimit(RateLimitHelpers.makeAssetKey(controller.LIMIT_PSM_DEPOSIT(), Optimism.SUSDS),  0, 0);
+        _assertRateLimit(RateLimitHelpers.makeAssetKey(controller.LIMIT_PSM_WITHDRAW(), Optimism.SUSDS), 0, 0);
```

Comment:
Align all the Optimism.USDC, USDS, and SUDS params as well

Reply:
aligned